app/views/devise/registrations/_new_senior.html.erb

```diff
@@ -48,22 +48,32 @@
                         input_html: { class: "seko-search-bar" } %>
             <%= f.input :pathology,
                         label: "Souffrez vous d'une pathologie chronique?",
-                        collection: ['Parkinson','Alzheimer', 'Autre'],
+                        collection: [
+                          ['Parkinson', 'Parkinson' ],
+                          ['Alzheimer', 'Alzheimer'],
+                          ['Autre', 'Autre']
+                        ],
```

Comment:
See comment in _new_family.html.erb about collection

Reply:
Arrays updated !